scripts/chroot.sh

```diff
@@ -945,9 +945,10 @@ cat > "${DIR}/chroot_script.sh" <<-__EOF__
 				passwd -l root || true
 			else
 				passwd <<-EOF
-				root
-				root
+				${rfs_root_password}
+				${rfs_root_password}
 				EOF
+				echo "export PATH=\$PATH:/usr/local/sbin:/usr/sbin:/sbin" >> /root/.bashrc
```

Comment:
Here am not sure this is valid for all images. At least for debian buster this helps

Reply:
last time i tried stretch, i think the path's were still messed up as root, so if it fixes buster, it should be good for stretch then...

Reply:
cool, thanks...